PSKoans/Koans/Cmdlets 1/AboutPSProviders.Koans.ps1

```diff
@@ -264,6 +268,9 @@ Describe 'Variable Provider' {
         }
 
         It 'exposes data from default variables' {
+            if (Test-Path -Path 'Variable:Test') {
+                Remove-Variable -Name Test
+            }
```

Comment:
This feels like an odd place for this to go. Does it still work if we remove the variable at the end of `allows access to variables in the current scope` instead?

Reply:
somehow after adding that code, it doesn't complain after adding number to first placeholder.  It's more like a workaround until we understand the real problem and can fix that

Reply:
maybe adding a comment about that code being workaround would help keeping the attention :)

Reply:
I also tried to add the same code to the end of previous It block, but it didn't help at all.  Maybe it isn't about scoping and is realated to something else.

OTOH, the same problem didn't occur in command line.

Reply:
Hmm, yeah. I'll take a closer look today and see if I can find out what's up with this one. 🙂

Reply:
Sorry, i misplaced the block, will fix that